WalletWasabi/WabiSabi/Backend/Banning/Inmate.cs

```diff
@@ -11,21 +11,12 @@ namespace WalletWasabi.WabiSabi.Backend.Banning
 	/// <summary>
 	/// A UTXO that's living inside the prison.
 	/// </summary>
-	public class Inmate
+	public record Inmate(
+		OutPoint Utxo,
+		Punishment Punishment,
+		DateTimeOffset Started,
+		Guid LastDisruptedRoundId)
 	{
-		public Inmate(OutPoint utxo, Punishment punishment, DateTimeOffset started, Guid lastDisruptedRoundId)
-		{
-			Utxo = utxo;
-			Punishment = punishment;
-			Started = started;
-			LastDisruptedRoundId = lastDisruptedRoundId;
-		}
-
-		public OutPoint Utxo { get; }
-		public Punishment Punishment { get; }
-		public DateTimeOffset Started { get; }
-		public Guid LastDisruptedRoundId { get; }
-
 		public TimeSpan TimeSpent => DateTimeOffset.UtcNow - Started;
 
 		public static Inmate FromString(string str)
```

Comment:
I would say that these `*String` variables does not bring additional clarity. One can probably inline it and the result is the same.